api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -30,11 +30,14 @@ message QuicProtocolOptions {
   core.v3.QuicProtocolOptions quic_protocol_options = 1;
 
   // Maximum number of milliseconds that connection will be alive when there is
-  // no network activity. 300000ms if not specified.
+  // no network activity.
+  //
+  // If it is less than 1ms, Envoy will use 1ms. 300000ms if not specified.
```

Comment:
Should the config be rejected in this case?
What I'm worried about is a user that inserts a small value, and is not aware that Envoy uses a different value.

Another question is whether there's a max value?

Reply:
There is no max value. This value shouldn't be too small anyway. Even 1ms is not desirable in real production deployment. If the user configured value is even less than 1ms, the expected behavior oddness would be about the same as 1ms. Would add a warning about the overriding be sufficient?